packages/@n8n/nodes-langchain/nodes/embeddings/EmbeddingsOpenAI/EmbeddingsOpenAi.node.ts

```diff
@@ -79,7 +79,7 @@ export class EmbeddingsOpenAi implements INodeType {
 			},
 		],
 		group: ['transform'],
-		version: 1,
+		version: [1, 1.1],
```

Comment:
Why increase the version-number?

Reply:
The new default embedding was already in place for this node, but not active because the version was not updated.